.github/workflows/build_wheels_macos.yml

```diff
@@ -36,7 +36,7 @@ jobs:
             cache-key: macos-ffmpeg
             pre-script: packaging/pre_build_script.sh
             post-script: packaging/post_build_script.sh
-            smoke-test-script: test/smoke_test/smoke_test.py
+            smoke-test-script: test/smoke_test/smoke_test_no_ffmpeg.py
```

Comment:
Should this be ``test/smoke_test/smoke_test.py --no-ffmpeg`` instead?

Reply:
It seems that you cannot pass an argument

https://github.com/pytorch/test-infra/blob/main/.github/workflows/build_conda_linux.yml#L176